docs/src/meshes/mesh_data_meshes.md

```diff
@@ -0,0 +1,125 @@
+# Unstructured simplicial meshes
+
+Support for simplicial meshes uses the [StartUpDG.jl](https://github.com/jlchan/StartUpDG.jl)
+package. These meshes are constructed by specifying a list of vertex coordinates `vertex_coordinates_x`,
+`vertex_coordinates_y`, `vertex_coordinates_z` and a connectivity matrix `EToV` where `EToV[e,:]`
+gives the vertices which correspond to element `e`.
+
+We make a few simplifying assumptions about simplicial meshes:
+* meshes consist of a single type of element
+* meshes are _conforming_ (e.g., each face of an element is shared with at most one other element).
+* the geometric mapping from reference to physical elements is polynomial (currently, only affine
+mappings are supported).
```

Comment:
```suggestion
* the geometric mapping from reference to physical elements is polynomial (currently, only affine
  mappings are supported).
```
Otherwise, it doesn't render correctly, see https://trixi-framework.github.io/Trixi.jl/previews/PR690/meshes/mesh_data_meshes/